gcloud-java-core/src/main/java/com/google/gcloud/RetryParams.java

```diff
@@ -206,47 +205,53 @@ public static RetryParams getDefaultInstance() {
     return DEFAULT_INSTANCE;
   }
 
+  /**
+   * Returns an instance of {@code RetryParams} where `retryMaxAttempts` and `retryMinAttempts` are
+   * set to `1`.
+   */
   public static RetryParams noRetries() {
     return NO_RETRIES;
   }
 
   /**
-   * Returns the retryMinAttempts.
+   * Returns the retryMinAttempts. Default value is {@link #DEFAULT_RETRY_MIN_ATTEMPTS}.
```

Comment:
should we use {@value #static_field}. e.g.  {@value #DEFAULT_RETRY_MIN_ATTEMPTS} instead of {@value #DEFAULT_RETRY_MIN_ATTEMPTS}.